src/components/apis/details-of-api/detailsOfApiContract.ts

```diff
@@ -1,9 +1,14 @@
 import { Contract } from "@paperbits/common";
 import { HyperlinkContract } from "@paperbits/common/editing";
+import { LocalStyles } from "@paperbits/common/styles";
 
 export interface DetailsOfApiContract extends Contract {
     /**
      * Link to a page that contains API changlog.
      */
     changeLogPageHyperlink?: HyperlinkContract;
+    /**
+     * Widget local styles.
+     */
+    styles: LocalStyles;
```

Comment:
Are styles here optional? if yes please do
```suggestion
    styles?: LocalStyles;
```